packages/api-bzz-react-native/src/index.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-env browser */
+// @ts-ignore
+import { URL } from 'universal-url'
+import {
+  BaseBzz,
+  BzzConfig,
+  DirectoryData,
+  UploadOptions,
+} from '@erebos/api-bzz-base'
+import { hexValue } from '@erebos/hex'
+
+export * from '@erebos/api-bzz-base'
+
+export class Bzz extends BaseBzz<Response> {
+  public constructor(config: BzzConfig) {
+    const { url, ...cfg } = config
+    super(window.fetch.bind(window), { ...cfg, url: new URL(url).href })
```

Comment:
Shouldn't this use `global` instead of `window`?

Reply:
Not sure if this is needed, fetch is available globally in react-native, should we just pass in `fetch`?

Reply:
I'm not sure in RN but on the Web it needed to be bound to the proper context, there was some issue without.